Std/Tactic/NormCast/Ext.lean

```diff
@@ -89,15 +89,14 @@ def classifyType (ty : Expr) : MetaM Label :=
     else if rhsHeadCoes < lhsHeadCoes then do
       return Label.squash
     else do
-      throwError
-          ("norm_cast: badly shaped shaped squash lemma, " ++
-        "rhs must have fewer head coes than lhs{indentExpr ty}")
+      throwError "norm_cast: badly shaped shaped squash lemma, \
+        rhs must have fewer head coes than lhs{indentExpr ty}"
 
 /-- The `push_cast` simp attribute. -/
 initialize pushCastExt : SimpExtension ←
-  registerSimpAttr `push_cast <|
-    "The `push_cast` simp attribute uses `norm_cast` lemmas " ++
-    "to move casts toward the leaf nodes of the expression."
+  registerSimpAttr `push_cast
+    "The `push_cast` simp attribute uses `norm_cast` lemmas \
+     to move casts toward the leaf nodes of the expression."
```

Comment:
```suggestion
  registerSimpAttr `push_cast "\
    The `push_cast` simp attribute uses `norm_cast` lemmas \
    to move casts toward the leaf nodes of the expression."
```
this one is fine either way, but in general I think using hanging indent this way makes it easier to line things up without awkward 3 space indent issues.